index.html

```diff
@@ -279,5 +279,33 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+                <li class="shape1"><img src="image/background/contact-top1.png"></li>
+                <li class="shape2"><img src="image/background/contact-bottom.png"></li>
+                <li class="shape3"><img src="image/background/Capture3.PNG"></li>
```

Comment:
- [ ] Please note that background images are added in the css file and not in the html. By defining background images in the CSS file, you can easily reuse them across multiple elements or pages within your website. 👍🏾 
I suggest you use the whole background image as one. do not pick parts of it and add them separately.